src/monodroid/config.xml

```diff
@@ -9,6 +9,7 @@
 	<dllmap wordsize="32" dll="libintl" target="/system/lib/libc.so" />
 	<dllmap wordsize="64" dll="libintl" target="/system/lib64/libc.so" />
 	<dllmap dll="MonoPosixHelper" target="libMonoPosixHelper.so" />
+	<dllmap dll="MonoPosixHelper" target="libmono-native.so" />
```

Comment:
...huh?  Why would we remap `MonoPosixHelper` to `libmono-native.so`?  Wouldn't this break `Mono.Posix.dll`?